tests/configuration.test.ts

```diff
@@ -38,41 +48,46 @@ describe("Configuration tests", () => {
         },
       } as unknown as GitHubContext<"issues.closed">["payload"],
       octokit: {
```

Comment:
why are there two `getContent` functions, one in `octokit.repos` and another in `octokit.rest.repos`?

Reply:
For the download you used the `rest` while in the rest of the code it's without it
https://github.com/ubiquity/ubiquibot-kernel/blob/development/src/github/utils/config.ts#L141

I guess I can change it, although it made tests easier in the end.

Reply:
that's my bad then. in #64 I changed [octokit plugin](https://github.com/ubiquity/ubiquibot-kernel/blob/development/src/github/github-client.ts) from `legacyRestEndpointMethods` to `restEndpointMethods` which uses only `rest`

Reply:
So moving on, should we only use `.rest` when using Octokit?

Reply:
yes, legacy way was using without `.rest`